libs/api-authentication/src/lib/config.ts

```diff
@@ -0,0 +1,4 @@
+import os from 'node:os'
+import path from 'node:path'
+
+export const CONFIG_DIRECTORY = path.join(os.homedir(), `.featureboard`)
```

Comment:
Would be good to leave this in the leaf libs.

Ideally libraries should not have side effects

Reply:
The idea is that you would still have to login with the cli regardless of the project you are running so having this central reduces the possibility of issues arising from mis configuration.